pulser/_seq_drawer.py

```diff
@@ -106,6 +108,7 @@ def phase_str(phi):
     # Boxes for qubit and phase text
     q_box = dict(boxstyle="round", facecolor='orange')
     ph_box = dict(boxstyle="round", facecolor='ghostwhite')
+    area_ph_box = dict(boxstyle='round', facecolor='ghostwhite')
```

Comment:
Also, I noticed you removed the box transparency. I think you should keep it, it's good to see the pulse behind

Reply:
Oh yes, it would be better to have pulse still shown behind the box. I have added  the `alpha` parameter.